docs/Rendszerterv.md

```diff
@@ -2,15 +2,19 @@
 
 ## 1. A rendszer célja
 - Leírja hogy mit szeretne megoldani a rendszer.
-- Példa: “A rendszer célja, hogy a felhasználó játékos körülmények között tud
-tanulni, különböző feladatokat megoldani. A felhasználó pontszámokat kap
-arról, hogyan sikerült megoldania a feladatokat. Fontos, hogy a felhasználó
+- A rendszer célja, hogy a felhasználó játékos körülmények között tud
```

Comment:
This was only an example, not the actual goal of the project. AMI is a publication site, not a playful learning application